lib/log.js

```diff
@@ -178,25 +178,59 @@ Lug.prototype.activityEvent = function (event, request, data) {
     return P.resolve()
   }
 
+  // log a flowEvent for all activityEvents
+  this.flowEvent(event, request)
+
+  var info = {
+    event: event
+  }
+
+  if (request.headers['user-agent']) {
+    info.userAgent = request.headers['user-agent']
+  }
+
+  optionallySetService(info, request)
+
+  Object.keys(data).forEach(function (key) {
+    info[key] = data[key]
+  })
+
+  this.logger.info('activityEvent', info)
+  this.statsd.write(info)
+
+  return P.resolve()
+}
+
+// Log a flow metrics event.
+// These events help understand the user flow.
```

Comment:
Specifically, a flow is a sign-in or sign-up user journey (which may or may not be completed).